docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences the commit latency.
 To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
+In contract, the `archive` should be using the disk with the largest throughput.
 
```

Comment:
I have no idea how to attache my comment on line 5 (so leaving it here where I can enter a comment)

In our standard docs on style has been to say "Storage Manager (SM)" and then use SM when referring to a Storage Manager later on a page. 

Thus, 
"The storage manager, SM for short,"
  becomes
"The Storage Manager (SM)"